cfgov/agreements/tests/test_legacy.py

```diff
@@ -132,13 +132,13 @@ def test_issuer_paging_frontend(self):
 
         path = reverse('issuer_search', kwargs={'issuer_slug': issuer.slug})
         resp = self.client.get(path)
-        self.assertTrue('page=2' in resp.content)
-        self.assertFalse('page=1' in resp.content)
+        self.assertTrue(b'page=2' in resp.content)
+        self.assertFalse(b'page=1' in resp.content)
 
         resp = self.client.get(path + '?page=2')
-        self.assertTrue('page=1' in resp.content)
-        self.assertFalse('page=2' in resp.content)
-        self.assertFalse('page=3' in resp.content)
+        self.assertTrue(b'page=1' in resp.content)
```

Comment:
Same suggestion as above:

```suggestion
        self.assertContains(resp, 'page=1')
        self.assertNotContains(resp, 'page=2')
        self.assertNotContains(resp, 'page=3')
```